### PR TITLE
#4554 fix Class not found exception in maven server

### PR DESCRIPTION
**Pull request by evidolob**
### What does this PR do?
Fixes `ClassNotFoundException` when download maven artifact sources.

### What issues does this PR fix or reference?
#4554

### CODE REVIEW

File: plugins/plugin-maven/maven-server/maven-server-impl/src/main/java/org/eclipse/che/maven/server/MavenServerImpl.java [hunk 1]
```diff
@@ -425,8 +425,9 @@ public MavenArtifact resolveArtifact(MavenArtifactKey artifactKey, List<MavenRem
                                                                              new ArtifactRequest(RepositoryUtils.toArtifact(artifact),
                                                                                                  remoteRepositories, null));
         return MavenModelUtil.convertArtifact(RepositoryUtils.toArtifact(artifactResult.getArtifact()), localRepository);
-        } catch (ArtifactResolutionException e) {
-            MavenServerContext.getLogger().info(e);
+        } catch (ArtifactResolutionException ignored) {
+            //we need ignore exception, it's some times has class that client doesn't has
```
Comment (skabashnyuk) [inline plugins/plugin-maven/maven-server/maven-server-impl/src/main/java/org/eclipse/che/maven/server/MavenServerImpl.java:429]: may be system.error.println()?

Comment (evidolob) [reply plugins/plugin-maven/maven-server/maven-server-impl/src/main/java/org/eclipse/che/maven/server/MavenServerImpl.java:429]: All sdtout/stderr of the maven sever is logged by client, but 'ArtifactResolutionException' is normal when maven can't find or download artifact, so I not sure that this exception need to be logged.

Comment (skabashnyuk) [reply plugins/plugin-maven/maven-server/maven-server-impl/src/main/java/org/eclipse/che/maven/server/MavenServerImpl.java:429]: up2you

**Review by vparfonov — APPROVED** [review]

**Review by dmytro-ndp — APPROVED** [review]
It definitely fixes issue #4554
screenshot from 2017-03-30 16 19 58

**Pull request merged**